lib/screens/SiteLogsScreen.dart

```diff
@@ -49,18 +51,29 @@ class _SiteLogsScreenState extends State<SiteLogsScreen> {
       scrollable: SimpleScrollable.both,
       scrollController: controller,
       onRefresh: () async {
-        await loadLogs();
+        await logsNotifier.loadLogs(logFile: widget.site.logFile);
         refreshController.refreshCompleted();
       },
       onLoading: () async {
-        await loadLogs();
+        await logsNotifier.loadLogs(logFile: widget.site.logFile);
         refreshController.loadComplete();
       },
       refreshController: refreshController,
       child: Container(
           padding: EdgeInsets.all(5),
           constraints: logBoxConstraints(context),
-          child: SelectableText(logs.trim(), style: TextStyle(fontFamily: 'RobotoMono', fontSize: 14))),
+          child: ListenableBuilder(
+            listenable: logsNotifier,
+            builder: (context, child) => SelectableText(
+                switch (logsNotifier.logsResult) {
+                  Ok<String>(:var value) => value.trim(),
+                  Error<String>(:var error) => error is LogsNotFoundException
+                      ? "No logs file found."
```

Comment:
Why not use the exception's text here, which is the same, but without a period at the end?

Reply:
Sounds good